recipes-omnect/omnect-device-service/omnect-device-service.inc

```diff
@@ -25,19 +25,22 @@ LIC_FILES_CHKSUM = " \
 
 DEPENDS += "\
     azure-iot-sdk-c \
+    modemmanager \
     pkgconfig-native \
 "
 
 RDEPENDS:${PN} += "\
     aziot-identityd\
     ca-certificates\
+    modemmanager \
+    modemmanager-dev \
```

Comment:
runtime dependency on a dev package? why?

Reply:
it's not necessary, has been dropped.